### PR TITLE
:sparkles: Pool details form manage slippage

### PR DESCRIPTION
**Pull request by KipandaJr**
---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206703665848189

### CODE REVIEW

**Review by MarioBatalha — COMMENTED** [review]
Screenshot 2024-03-05 at 21 11 06

Fix the form in this breakpoint

File: views/swap/components/select-token-modal/not-found.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  <Box
+    p="4xl"
+    gap="xl"
+    flex="1"
+    color="text"
+    display="flex"
+    overflowY="auto"
+    alignItems="center"
+    flexDirection="column"
+    bg="surface.containerLow"
```
Comment (git-marcopitra) [inline views/swap/components/select-token-modal/not-found.tsx:16]: verify if this color exist

File: views/swap/components/select-token-modal/select-token-modal-body.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+} from './select-token-modal.types';
+
+const SelectTokenModalBody: FC<SelectTokenModalBodyProps> = ({
+  control,
+  handleSelectToken: onSelectToken,
+}) => {
+  const { network } = useNetwork();
+  const { coins, isFetchingCoinBalances } = useWeb3();
+  const favoriteTokenTypes = useReadLocalStorage<ReadonlyArray<string>>(
+    `${LOCAL_STORAGE_VERSION}-sui-coins-${network}-favorite-tokens`
```
Comment (git-marcopitra) [inline views/swap/components/select-token-modal/select-token-modal-body.tsx:27]: `sui-coins` to `movement`

File: views/swap/components/select-token-modal/token-modal-item.tsx [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+  symbol,
+  onClick,
+  selected,
+}) => {
+  const { network } = useNetwork();
+  const { colors } = useTheme() as Theme;
+  const [isLoading, setLoading] = useState(false);
+  const [favoriteTokens, setFavoriteTokens] = useLocalStorage<
+    ReadonlyArray<string>
+  >(`${LOCAL_STORAGE_VERSION}-sui-coins-${network}-favorite-tokens`, []);
```
Comment (git-marcopitra) [inline views/swap/components/select-token-modal/token-modal-item.tsx:30]: `sui-coins` > `movement`

**Review by git-marcopitra — CHANGES_REQUESTED** [review]

File: views/swap/manage-slippage/manage-slippage-form/index.tsx [hunk 4]
```diff
@@ -36,19 +31,21 @@ const ManageSlippageForm: FC<ManageSlippageProps> = ({ handleManageView }) => {
...
           Slippage Tolerance
         </Typography>
         <Box
           gap="xs"
           flexDirection="column"
-          gridTemplateColumns="4fr repeat(3, auto)"
           display={['flex', 'flex', 'flex', 'grid']}
+          gridTemplateColumns="4fr repeat(3, auto)"
         >
           <TextField
```
Comment (MarioBatalha) [inline views/swap/manage-slippage/manage-slippage-form/index.tsx:45]: Screenshot 2024-03-05 at 21 11 06


Fix form alignment

**Review by MarioBatalha — CHANGES_REQUESTED** [review]

**Review by MarioBatalha — APPROVED** [review]

**Review by git-marcopitra — APPROVED** [review]

**Review by atilasousa — APPROVED** [review]

**Pull request merged**